### PR TITLE
MRC-1280 Directions overview (single floor) demo

### PR DESCRIPTION
**Pull request by vercasson**
- [x] This change is documented in CHANGELOG.md

I'll update the changelog after the other ticket is merged and I pull those updates in

### CODE REVIEW

File: src/web-sdk.tsx [hunk 1]
```diff
@@ -707,7 +749,7 @@ export class API {
    */
   async fetchPlacemarksByLocation(locationID: string): Promise<FloorData[]> {
     if (!locationID) {
-      requiredParam("fetchPlacemarksByFloor", "locationID");
+      requiredParam("fetchPlacemarksByLocation", "locationID");
```
Comment (vercasson) [inline src/web-sdk.tsx:752]: oopsie

**Review by tarakreft — APPROVED** [review]
LGTM

File: demo/index.html [hunk 2]
```diff
@@ -202,11 +196,21 @@ <h3><a href="container-styling">Container styling</a></h3>
...
+      <p>A demo of how to create a dark mode theme for the SDK.</p>
+
       <h3><a href="advanced-styling">Advanced styling</a></h3>
       <p>An advanced styling demo.</p>
 
-      <h3><a href="dark-theme">Dark theme</a></h3>
-      <p>A demo of how to create a dark mode theme for the SDK.</p>
+      <h3>
+        <a href="directions-overview/"
+          >[BETA] Directions Overview (single floor)</a
```
Comment (amyhliao) [inline demo/index.html:207]: Is this where we would add the support email for consumers to submit their input on the beta that was mentioned during the Jamboree session??

**Review by amyhliao — APPROVED** [review]

File: demo/index.html (hunk 2 above)
Comment (vercasson) [reply demo/index.html:207]: @amyhliao Good memory! have a note regarding that. I think we'll add that to the demo itself.

**Pull request merged**